src/Sarif.Converters/ToolFormatConverter.cs

```diff
@@ -100,12 +107,12 @@ public class ToolFormatConverter
         // Set up a Chain of Responsibility that will get the converter from the first
         // factory capable of creating it.
         // This method is internal, rather than private, for test purposes.
-        internal static ConverterFactory CreateConverterFactory(string pluginAssemblyPath)
+        internal ConverterFactory CreateConverterFactory(string pluginAssemblyPath)
         {
             ConverterFactory factory = new BuiltInConverterFactory();
             if (!string.IsNullOrWhiteSpace(pluginAssemblyPath))
             {
-                factory = new PluginConverterFactory(pluginAssemblyPath)
+                factory = new PluginConverterFactory(pluginAssemblyPath, this.assemblyLoadFileDelegate)
```

Comment:
>this. [](start = 73, length = 5)

Should the 'this' keyword be present outside of the constructor for this variable?

Reply:
See my reply above.